### ISSUE TITLE
How to train it ?

### ISSUE THREAD

**Issue by maple0leaves** [fengyuewuya/EquiPocket#3]
Hello, I read your paper and looked at the code，your work is very good .

And I want to know what the input of loss is？
Could you provide training code or give some tips on how I should train it?
After reading the paper, I still don’t know what data should be input into Equipocket during training. Is it protein.pdb or pocket.pdb or ligand.mol2?

I would be grateful if you could give some tips，Thank you !
